resources/views/celebrities/create.blade.php

```diff
@@ -0,0 +1,49 @@
+<x-app-layout>
+    <x-slot name="header">
+        Create Celebrity  
+    </x-slot>
+        <form action="{{ route('celebrities.store')}}" method="POST">
+            @csrf
+            This is to create a new celebrity entry point
+            <div>
+                Name: 
+                <input
+                    type="text"
+                    name="name"
+                    id="name"
+                    value= "{{ old('name') }}"
+                >
+            </div>
+            <div>
+                D.O.B: 
+                <input
+                    type="date"
+                    name="dob"
+                    id="dob"
+                    value="{{ old('dob') }}"
+                >
+            </div>
+            <div>
+                Nationality: 
+                <input
+                    type="text"
+                    name="nationality"
+                    id="nationality"
+                    value= "{{ old('nationality') }}"
+                >
+            </div>
+            <div>
+                Bio: 
+                <input
+                    type="text"
+                    name="bio"
+                    id="bio"
+                    value= "{{ old('bio') }}"
```

Comment:
With longer texts input I would suggest to use a textarea instead of the input. 

Also, try to use the components that are available to you here ;D You can take a look at the login and see how they used the `input` and the `label` components.